app/components/GithubActionsScheduledJobs.tsx

```diff
@@ -0,0 +1,64 @@
+import Image from "next/image";
+
+function GithubActionsJob({
+  repo,
+  workflowName,
+}: {
+  repo: string;
+  workflowName: string;
+}) {
+  return (
+    <div>
+      <a
+        href={`https://github.com/ISISComputingGroup/${repo}/actions/workflows/${workflowName}`}
+        className={
+          "flex-row flex items-center content-center h-10 w-full text-center rounded-lg bg-gray-800 border-2 border-black hover:border-white"
+        }
+      >
+        <h2 className={"text-center text-lg w-1/2"}>{repo}: </h2>
+        <Image
+          src={`https://github.com/ISISComputingGroup/${repo}/actions/workflows/${workflowName}/badge.svg`}
```

Comment:
When we had the beam status image we had some trick to auto-refresh it - did we want to do that, or happy that because these are just changing once per night we'll manually refresh the page?

Reply:
think im happy that we just refresh the page - probably don't need the added complexity of a refresh especially because if you forget to clear the timeout afterwards you end up with a nice memory leak